services/src/northbound-service/nb-api/src/main/java/org/openkilda/northbound/dto/v1/switches/SwitchDto.java

```diff
@@ -42,4 +42,19 @@
 
     @JsonProperty("under_maintenance")
     private boolean underMaintenance;
+
+    @JsonProperty(value = "of_version")
```

Comment:
nit: I think it's better to remove "value" to be consistent with other fields

Reply:
Fixed.